src/components/Activity/ActivityList.js

```diff
@@ -11,7 +11,7 @@ import {
 import ActivityItem from './ActivityItem'
 import { useActivity } from '@providers/ActivityProvider'
 
-import { ACTIVITY_STATUS_PENDING } from './activity-statuses'
+import { ActivityStatus } from './activity-statuses'
```

Comment:
same here

Reply:
Same comment as above, when you have several constants, you should use a enum.